apps/frontend/src/app/components/map-list/map-list-item.component.ts

```diff
@@ -48,10 +48,18 @@ export class MapListItemComponent implements OnChanges {
   prefix: string;
   tierStyle: string;
 
+  protected isLoggedIn: boolean;
+
   constructor(
     private readonly localUserService: LocalUserService,
     private readonly messageService: MessageService
-  ) {}
+  ) {
+    this.isLoggedIn = this.localUserService.isLoggedIn();
```

Comment:
Rather than handling subscription yourself here, you can pipe the localusersubject changes, map to whether the user is logged in, then rather than call subscribe explicitly, use an async pipe.

Reply:
To clarify, use the LocalUserService.LoginChanged BehaviourSubject<bool>, then expose the service. Then, in the template, you can do e.g. `@if (luService.LoginChange | async)`. I don't use async pipes enough, but since we have so much state attached to a single service, it's a very good case for them.